ts/components/inputs/number.ts

```diff
@@ -0,0 +1,47 @@
+import { Entangle } from "@/components/alpine"
```

Comment:
This file is not following the project code style, use the `eslint` to format the code

Reply:
Done!